content/02.intro.md

```diff
@@ -1,10 +1,12 @@
 ## Introduction {#intro}
 
 Deep learning (DL), a subfield of machine learning (ML) implementing artificial neural networks with many layers, is increasingly used for the analysis of biological data [@doi:10.1098/rsif.2017.0387].
-Despite its growing popularity, DL itself remains an active area of research. Its everchanging complexity and lack of current beginner resources focused on biological applications pose large barriers of entry to newcomers who wish to utilize state-of-the-art DL in their research.
-Biological insight garnered from DL has been well-documented in the scientific literature, with applications ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z]. However, few resources articulate DL best practices to the scientific community. Most instructional literature focuses on ML broadly, rather than DL specifically, further limiting accessibility and reproducibility [@doi:10.1186/s13040-017-0155-3].
+There are many well-documented cases in the literature of novel biological insight being revealed by careful application of DL methods ranging from predicting protein-drug binding kinetics [@doi:10.1038/s41467-017-02388-1] to identifying the lab-of-origin of synthetic DNA [@doi:10.1038/s41467-018-05378-z]. 
+However, a lack of relevant up-to-date resources focused on biological applications of DL pose a significant barrier to newcomers wishing to utilize state-of-the-art DL in their research.
+As DL is an active and specialised research area, resources are rapidly rendered obsolete and few resources articulate DL best practices to the scientific community.
+Most instructional literature focuses on ML broadly, rather than DL specifically, further limiting accessibility and reproducibility [@doi:10.1186/s13040-017-0155-3].
```

Comment:
To be a bit more specific, since there's tons and tons of instructional DL literature out there, maybe we should say that we refer to instructional literature concerning applications of ML in the context of biology. Also, it sounds a little bit like criticizing the cited article for not discussing DL. Maybe this was not the aim of this article?! I think we should refer to this articles (and similar articles) in a less negative way.

Reply:
@fmaguire can you please make revisions to address this comment so we can merge this pull request?

I agree with @rasbt's comment above.  One option would be to move the `@doi:10.1186/s13040-017-0155-3` reference to the next parpagraph, either as a citation for `the importance of understanding and applying ML fundamentals` or elsewhere.  This may help show that are supportive of the recommendations in that article and want to build on them for the special considerations of DL.

I realize this sentence is not part of your current updates, but this may be a good time to make the edits anyway.

Reply:
I agree with @agitter 's suggestion

Reply:
One thing that I want to make sure we get across in the introduction is that we're not trying to provide a detailed DL tutorial (which will be out of date in ~0.001 seconds given the rate of DL development) but rather to articulate *general* best practices.

Maybe "resources are rapidly rendered obsolete and few resources articulate DL best practices" -> "detailed resources are rapidly rendered obsolete and few resources articulate general DL best practices"

Reply:
Sorry, got swamped by other projects. Changes made as requested.